cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -497,6 +527,17 @@ open class Node @Inject constructor(private val project: Project) {
         }
     }
 
+    internal fun installDriver(driverGroup: String, driverName: String, driverVersion: Any?) {
```

Comment:
There's also `installDrivers` below that seems to do something very similar. Can we reuse that by appending the driver to the drivers configuration instead?

Reply:
Removed now